qiskit/algorithms/evolvers/variational/var_qite.py

```diff
@@ -0,0 +1,110 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Variational Quantum Imaginary Time Evolution algorithm."""
+from typing import Optional, Union, Type, Callable
+
+import numpy as np
+from scipy.integrate import OdeSolver
+
+from qiskit.opflow import ExpectationBase
+from qiskit.algorithms.evolvers.imaginary_evolver import ImaginaryEvolver
+from qiskit.utils import QuantumInstance
+from .solvers.ode.forward_euler_solver import ForwardEulerSolver
+from .variational_principles import ImaginaryVariationalPrinciple
+from .var_qte import VarQTE
+
+
+class VarQITE(VarQTE, ImaginaryEvolver):
+    """Variational Quantum Imaginary Time Evolution algorithm.
+
+    .. code-block::python
+
+        from qiskit.algorithms import EvolutionProblem
+        from qiskit.algorithms import VarQITE
+        from qiskit import BasicAer
+        from qiskit.circuit.library import EfficientSU2
+        from qiskit.opflow import SummedOp, I, Z, Y, X
+        from qiskit.algorithms.evolvers.variational.variational_principles.
+        imaginary_mc_lachlan_principle import (
+            ImaginaryMcLachlanPrinciple,
+        )
```

Comment:
Since I noticed this in passing this is another instance of importing as I mentioned earlier - I am sure a search will be more reliable than me noting any further ones though!